src/components/molecules/UserInfo/UserInfo.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import { Icon, Text } from '../..';
+
+import './UserInfo.scss';
+
+export const UserInfo = ({ date, city, hobbies }) => {
+    return (
+        <div className='molecule__user__info'>
+            <Text type='primary' size='extraLarge'>User Details</Text>
+            <div>
+                <Icon iconName='birthday' size='medium' />
+                <Text type='primary' size='large'>{date}</Text>
```

Comment:
date that we would gate from API would be in UTC ISO8 format I guess, we have to format that date